dgraph/sample/schema.gql

```diff
@@ -1,13 +1,20 @@
-type Entity @dgraph(type: "Entity") {
-    type: String! @dgraph(pred:"type") @search(by: [exact])
-    code: String @dgraph(pred:"code") @search(by: [exact])
-    description: String! @dgraph(pred:"description") @search(by: [fulltext, exact])
-    properties: [Property] @dgraph(pred:"has_property")
-    children: [Entity] @dgraph(pred:"has_child")
+type Entity {
```

Comment:
Is this a duplicate of the Apollo schema?

Reply:
Yeah, they are pretty close (at the moment the Entity and Property nodes are 1-1 directly mapped), but I think we still need it to create the Dgraph schema when setting up the DB? 

The alternative is to use a single schema file to set up both Dgraph and Apollo, then Apollo will need a separate script/routine to add in the additional fields we need (for interactions/custom queries etc).

Reply:
My mistake, thought this was the old GraphQL schema (not the Dgraph schema)!

I think we can probably move this out of the `sample/` dir now (but can do that in another PR maybe).

Reply:
That's a good point - it's not really a sample anymore is it? :D